### PR TITLE
feat: Remove all open-sauced/open-sauced workflows

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [x] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR updates the workflow files to use the new repository name "open-sauced/hot". 

_Generated using [OpenSauced](https://opensauced.ai/)._

## Related Tickets & Documents

https://github.com/open-sauced/open-sauced/issues/1491
## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
We have a few other repos that are still using this, but we can update them when we actually send it updates. With this PR open, I will go ahead and archive.

File: .github/workflows/maintenance.yml [hunk 1]
```diff
@@ -2,9 +2,18 @@ name: Maintenance
 
 on:
   schedule:
-    - cron: '0 3 * * *'
+    - cron: '0 2 * * *'
```
Comment (bdougie) [inline .github/workflows/maintenance.yml:5]:
```suggestion
    - cron: '0 3 * * *'
```

File: .github/workflows/maintenance.yml [hunk 2]
```diff
@@ -4,7 +4,16 @@ on:
   schedule:
     - cron: '0 3 * * *'
   workflow_dispatch:
+  workflow_call:
```
Comment (bdougie) [inline .github/workflows/maintenance.yml:7]:
```suggestion
```

**Comment by bdougie** [comment]
Turns out all workflows still work even though it's now archived

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**